core/capabilities_test.go

```diff
@@ -76,6 +79,45 @@ func TestCapability_CompatibleBitstring(t *testing.T) {
 	})
 }
 
+// We need this in order to call `NvidiaTranscoder::Transcode()` properly
+func setupWorkDir() (string, func()) {
+	tmp, _ := ioutil.TempDir("", "")
+	WorkDir = tmp
+	cleanup := func() {
+		WorkDir = ""
+		defer os.RemoveAll(tmp)
+	}
+	return tmp, cleanup
+}
+
+func TestCapability_TranscoderCapabilities(t *testing.T) {
+	tmpdir, cleanup := setupWorkDir()
+	defer cleanup()
+
+	// nvidia test
+	devices, err := common.ParseNvidiaDevices("all")
+	devicesAvailable := err != nil && len(devices) > 0
```

Comment:
maybe missed an equal symbol
```suggestion
	devicesAvailable := err == nil && len(devices) > 0
```

Reply:
good catch!